File: js/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const nome = lines.shift();
const salarioFixo = Number(lines.shift());
const totalVendas = Number(lines.shift());
const salarioComBonus = salarioFixo + (totalVendas * 0.15);
console.log(`TOTAL = R$ ${salarioComBonus.toFixed(2)}`);

File: cs/1009.cs
string nome = Console.ReadLine();
double salarioFixo = double.Parse(Console.ReadLine());
double totalVendas = double.Parse(Console.ReadLine());
double salarioComBonus = salarioFixo+(totalVendas*0.15);
Console.WriteLine($"TOTAL = R$ {salarioComBonus.ToString("0.00")}");